src/vectorchord/usage/search.md

```diff
@@ -0,0 +1,72 @@
+# Search
+
+Get the nearest 5 neighbors to a vector
+
+```sql
+SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
+```
+
+## Operators
+
+These operators are used for distance metrics:
+
+| Name  | Description                |
+| ----  | -------------------------- |
+| <->   | squared Euclidean distance |
+| <#>   | negative dot product       |
+| <=>   | cosine distance            |
+
+## Filter
+
+For a given category, get the nearest 10 neighbors to a vector
+```sql
+SELECT 1 FROM items WHERE category_id = 1 ORDER BY embedding <#> '[0.5,0.5,0.5]' limit 10
+```
+
+## Query options
+
+#### `vchordrq.probes`
+    
+- Description: This GUC parameter `vchordrq.probes` controls how the vector space assists in query pruning. The more probes, the more accurate the search, but also the slower it is.
+- Type: integer
+- Default: ``
+- Example:
+    - `SET vchordrq.probes = 1` means that only one probe is used.
+    - `SET vchordrq.probes = 10` means that ten probes are used.
+- Note: The default value is `1`, which means that only one probe is used. This is the fastest search, but also the least accurate. If you want to improve the accuracy of the search, you can increase the number of probes. However, this will also slow down the search.
```

Comment:
```suggestion
- Note: The default value is ` `, which means that only one probe is used. This is the fastest search, but also the least accurate. If you want to improve the accuracy of the search, you can increase the number of probes. However, this will also slow down the search.
```